### DISCUSSION TITLE
Set voice channel

### DISCUSSION THREAD

**Discussion by Nuckerr** [jagrosh/MusicBot#697] [Questions] [answered]
What do I put for the `!setvc <voicechannel>` for the `<voicechannel>`. I tried the id but it didnt work so im not really sure

**Comment by ceocharley** [comment, marked as answer]
Hello Nucker!

To use the `!setvc` you follow the command up with the voice channel **name**, not voice channel id!
Glad I can help you out!
lokiiyyexe
